### PR TITLE
GPII-2802 - Include cloud-admin group to set up permissions

### PR DESCRIPTION
**Pull request by amatas**
Include the set up of the permissions of the group cloud-admin@. This code will detect a lack of permissions in the group as the same way that we did with the common SA. The rake command name has changed to `fix_organization_permissions` because we are setting the permissions of the common SA and the cloud-admin group.

### CODE REVIEW

File: shared/rakefiles/xk_infra.rake [hunk 1]
```diff
@@ -1,4 +1,4 @@
-organizations_permissions = [
+commonsa_organization_permissions = [
```
Comment (mrtyler) [inline shared/rakefiles/xk_infra.rake]: How about `common_sa`?

File: shared/rakefiles/xk_infra.rake [hunk 2]
```diff
@@ -9,6 +9,13 @@ organizations_permissions = [
   "roles/storage.admin"
 ]
 
+cloudadmin_organization_permissions = [
```
Comment (mrtyler) [inline shared/rakefiles/xk_infra.rake]: `cloud_admin_`?

File: shared/rakefiles/xk_infra.rake [hunk 3]
```diff
@@ -100,13 +107,15 @@ task :apply_common_infra => [@gcp_creds_file] do
...
   service_account = "serviceAccount:projectowner@#{ENV["TF_VAR_project_id"]}.iam.gserviceaccount.com"
   permissions_list.each do |permission|
-    organizations_permissions.delete(permission["role"]) if organizations_permissions.include?(permission["role"]) and permission["members"].include?(service_account)
+    commonsa_organization_permissions.delete(permission["role"]) if commonsa_organization_permissions.include?(permission["role"]) and permission["members"].include?(service_account)
+    cloudadmin_organization_permissions.delete(permission["role"]) if cloudadmin_organization_permissions.include?(permission["role"]) and permission["members"].include?("group:cloud-admin@raisingthefloor.org")
   end
-  raise "The common Service Account #{service_account} doesn't have the proper permissions #{organizations_permissions}.
 
-  An operator with admin privileges on the organization must run the following command: rake fix_common_service_account_permissions.
+  raise "The common Service Account #{service_account} or the cloud-adming group don't have the proper permissions.
```
Comment (mrtyler) [inline shared/rakefiles/xk_infra.rake]: cloud-admin

File: shared/rakefiles/entrypoint_common.rake [hunk 4]
```diff
@@ -7,12 +7,12 @@ task :apply_common_infra => [:set_vars, :configure_aws_restore] do
   sh "#{@exekube_cmd} rake apply_common_infra"
 end
 
-desc "[ONLY ADMIN] Fix the permissions for the common service account"
-task :fix_common_service_account_permissions => [:set_vars] do
-  # This task sets the permissions of the common service account for creating the
+desc "[ONLY ADMIN] Fix the permissions for admin accounts in the organization"
+task :fix_organization_permissions => [:set_vars] do
```
Comment (stepanstipl) [inline shared/rakefiles/entrypoint_common.rake]: What about calling this task `setup_...`? Setup to me sounds like something that you do initially, fix sounds more like something you do to repair something broken

Comment (amatas) [reply shared/rakefiles/entrypoint_common.rake]: mmm, well it's the command that rake proposes you when it detects that the permissions are wrong in the IAM organization (fix). Also, is a step that you have to do initially when a new organization is created (setup).

Either way works for me :dancing_men:

**Pull request merged**